gui/wxpython/main_window/frame.py

```diff
@@ -87,7 +92,7 @@ def __init__(
         id=wx.ID_ANY,
         title=None,
         workspace=None,
-        size=globalvar.GM_WINDOW_SIZE,
+        size=wx.Display(0).GetGeometry().GetSize(),
```

Comment:
This should be `wx.Display().GetGeometry()` which gives you the wx.Rect of the display the application was started on.

But `wx.DisplaySize()` gives me (3520, 1080) when I have 2 displays.

Reply:
Also you shouldn't need this here since you Maximize the window later, no?

Reply:
Ok. I already get it. :-)  We do not have to solve different sizes of 2 displays at all.